articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+
+### Verify machine properties
+
+Before you run a test failover, verify the machine properties, and make sure that they comply with Azure requirements. You can view and modify properties as follows:
+
+1. In Protected Items, select Replicated Items > VM.
+
+2. In the Replicated item pane, there's a summary of VM information, health status, and the latest available recovery points. Select Properties to view more details.
+
+3. In Compute and Network, modify settings as needed.
+
+    * You can modify the Azure VM name, resource group, target size, availability set, and managed disk settings.
+
+    * You can also view and modify network settings. These include the network/subnet to which the Azure VM is joined after failover, and the IP address that will be assigned to the VM.
+
+4. In Disks, view information about the operating system and data disks on the VM.
+
+### Run a test failover
+
+When you run a test failover, the following happens:
+
+1. A prerequisites check runs to make sure all of the conditions required for failover are in place.
+
+2. Failover processes the data using the specified recovery point:
+
+    * Latest processed: The machine fails over to the latest recovery point processed by Site Recovery. The time stamp is shown. With this option, no time is spent processing data, so it provides a low RTO (recovery time objective).
+
+    * Latest app-consistent.The machine fails over to the latest app-consistent recovery point.
+
+    * Custom. Select the recovery point used for failover.
+
+3. An Azure VM is created using the processed data.
+
+4. Test failover can automatically clean up Azure VMs created during the drill.
+
+### Run a test failover for a VM
+
+1. In **Protected items** > **Replicated Items**, select **VM** then **Test Failover**.
+
+2. For this walkthrough, we'll select to use the Latest processed recovery point.
+
+3. In Test Failover, select the target Azure network.
+
+4. Select OK to begin the failover.
+
+    ![Azure replication failover](images/azs-browser-recovery-failover.png)
+
+5. Track progress by selecting the VM to open its properties. Or, select the **Test Failover Job** in vault name > Settings > Jobs >Site Recovery jobs.
+
+    ![Azure replication test failover](images/azs-browser-recovery-testfailover.png)
+
+6. After the failover finishes, the replica Azure VM appears in the Azure portal > Virtual Machines. Check that the VM is the appropriate size, connected to the right network, and running.
+
+7. You should now be able to connect to the replicated VM in Azure.
+
+    1. To connect to the VM via RDP you will need to assign a public IP to the VM.
+
+    2. Navigate to the VMs **Network interface**.
+
+    3. Under settings select **IP configurations**, then select the config.
+
+    4. Enable **Public IP address**.
+
+    5. Configure **IP address** and create a new IP and select **OK**.
+
+    6. Select save to create the public IP for the VM.
```

Comment:
Click **Save** to create the public IP for the VM.

Reply:
done